### ISSUE TITLE
DIST_PACKAGE_CORRUPTED

### ISSUE THREAD

**Issue by fauxlefam** [veracrypt/VeraCrypt#789]
Hi there

I could successfully build the project however whenever I try to run the VeraCrypt.exe file after the setup I am seeing the following message:

[?]DIST_PACKAGE_CORRUPTED[?]

from my research I understand this is due an certificate error. I tried installing the certificates provided inside the signing folder without any luck. I tried the certificate provided [here](https://github.com/veracrypt/VeraCrypt/issues/405#issuecomment-683807415) and I am still not getting anywhere. 

I feel like I'm missing something obvious but I'm not quite sure what it is? can anybody point me to the right direction?

I am building on a windows 7 Ultimate OS using Visual Studio 2010 PRO w/ SP1 and running the setup on a windows 10 pro os.

There is an error during the signing process something about arm64 not being found but since I dont plan to use this I am simply ignoring this error. this cant be the cause now can it?

![Untitled](https://user-images.githubusercontent.com/17791714/117588584-11535d80-b0d9-11eb-9c5f-c2f01aaa7d0c.jpg)

**Comment by 827Dream** [comment]
hi，do you solve it ? how?

**Comment by Roby-P** [comment]
Resolved:
I have selected the VerarCrypt.exe file from folder  "C:\Program Files\VeraCrypt"
In property I have selected the label Digital signatures.
In the signature list, I selected IDRAX SARL and pressed the button below Details.
From the next screen I selected the "View Certificate" button.
I then installed the certificate by pressing the Install Certificate button.
I have installed it on my local computer by placing it in the "Trusted Authors" folder.
Subsequently, the problem disappeared and everything started working regularly again. 
![image](https://user-images.githubusercontent.com/102365159/160124734-c3e1f456-4654-4fe6-8aa9-d7693d3c97d7.png)
